src/index.ts

```diff
@@ -1,53 +1,48 @@
 import { ApiPromise, WsProvider } from "@polkadot/api";
-import { Header, SignedBlock } from "@polkadot/types/interfaces";
 import "@polkadot/api-augment";
 import * as PromClient from "prom-client"
 import * as http from "http";
 import { config } from "dotenv";
 import BN from "bn.js";
-import { xxhashAsHex } from "@polkadot/util-crypto";
-import * as winston from 'winston'
-import { AccountId, Balance, } from "@polkadot/types/interfaces/runtime"
-import { PalletBagsListListNode, PalletBagsListListBag } from "@polkadot/types/lookup"
 import { ApiDecoration } from "@polkadot/api/types";
+import parachains from "./parachains.json";
+import parachainsids from "./parachains-ids.json";
+import { exit } from "process";
+import {SystemExporter} from "./exporters/system";
+import {BalancesExporter} from "./exporters/balances";
+import {XCMTransfersExporter} from "./exporters/xcmTransfers";
+import {StakingMinerAccountExporter} from "./exporters/stakingMinerAccount";
+import {TransactionPaymentExporter} from "./exporters/transactionPayment";
+import {StakingExporter} from "./exporters/staking";
+import {PalletsMethodsExporter} from "./exporters/palletsMethodsCalls";
```

Comment:
You should aggregate all of these in `exporters/index.ts` to simplify all of these imports.

Reply:
yes, done